src/ensemble_scheduler/ensemble_scheduler.h

```diff
@@ -65,6 +66,13 @@ struct EnsembleInfo {
 
   bool is_decoupled_;
 
+  bool is_cache_enabled_;
+
+#ifdef TRITON_ENABLE_STATS
+  uint64_t ensemble_start_ns;
+  uint64_t ensemble_end_ns;
```

Comment:
Add `_` suffix for consistency with other member variables
```suggestion
  uint64_t ensemble_start_ns_;
  uint64_t ensemble_end_ns_;
```